### PR TITLE
Created the tutorials.rst file

### PR DESCRIPTION
**Pull request by risehr**
Added a user guide for tutorials in tutorials.rst file
Updated components.rst file

### CODE REVIEW

File: docs/tutorials.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    $ sudo apt-get install scrot
+    $ sudo apt-get install python3-tk
+    $ sudo apt-get install python3-dev
+    $ sudo apt-get install libx11-dev libxext-dev libxfixes-dev libxi-dev
+
+
+Now, just go into the _../MSS/tutorials/_ directory ::
+
+    $ cd ../MSS/tutorials/
+
```
Comment (ReimarBauer) [inline docs/tutorials.rst:87]: export PYTHONPATH=Path Of MSS

File: docs/tutorials.rst [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+Getting Started..
+=================
+
+On the Anaconda terminal, type the following ::
+
+ cd ..../MSS/$
+ $ conda activate mssdev
+
+
+ (mssdev)$ conda install --file requirements.d/tutorials.txt
```
Comment (ReimarBauer) [inline docs/tutorials.rst]: probably it is faster to use here mamba, same syntax as conda

File: docs/tutorials.rst [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+$ ./highlight-pointer -r 10
+
+This will make your cursor highlighted as a red cursor having radius as 10. You can minimize the terminal and continue
+your work with the highlighted cursor.
+When you have to stop just open that terminal again and type **CTRL + C**
+
+For more details in its customization, you can visit https://github.com/swillner/highlight-pointer
+
+THE RUNNING OF THE SCRIPTS
```
Comment (ReimarBauer) [inline docs/tutorials.rst]: no capitals please

File: docs/tutorials.rst [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+
+Now, just go into the _../MSS/tutorials/_ directory ::
+
+    $ cd ../MSS/tutorials/
+
+
+Keep the following things in mind before running a script
+
+* You should have only an **US keyboard layout**. If you have a different keyboard layout, you just need to change it to
```
Comment (ReimarBauer) [inline docs/tutorials.rst]: this should be on a more prominent place. e.g. before Getting Started

System requirements:
- full hd resolution 
- us keyboard configured

I it is also required to have a full HD resolution, not more, not less

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
a few things to change

File: docs/tutorials.rst (hunk 4 above)
Comment (risehr) [reply docs/tutorials.rst]: Okay, getting these things corrected.

**Comment by risehr** [comment]
Updated the tutorials.rst file and added some text files.

File: tutorials/textfiles/tutorial_remotesensing.txt [hunk 5]
```diff
@@ -0,0 +1,33 @@
+This is the REMOTE SENSING SECTION of the TOP VIEW. It shows the position and angle of the flight from any particular object in the sky.
+
+AZIMUTH is the forward direction line of the flight. If we go above tHE AZIMUTH, angle is in positive, and if we descend down, angle is negative.
+
+The TOP VIEW will be used to see what REMOTE SENSING is. From SELECT TO OPEN CONTROL, we select- REMOTE SENSING.
+
+Our main objective here is to find our the disturbance in visibility due to light source from any iluminating object like sun.
```
Comment (ReimarBauer) [inline tutorials/textfiles/tutorial_remotesensing.txt]: out?

File: tutorials/textfiles/tutorial_remotesensing.txt [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+The solar angle colour in dark to light is 0 to15 for RED and 15 to 45 for violet and 45-180 for green.
+
```
Comment (ReimarBauer) [inline tutorials/textfiles/tutorial_remotesensing.txt:10]: to 15

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
a few typos, see comments

**Comment by risehr** [comment]
Fixing them....

**Comment by risehr** [comment]
Fixed the typos in the text files.

**Review by ReimarBauer — APPROVED** [review]
Thx

**Comment by risehr** [comment]
Added the tutorial_performancesettings.txt file

**Comment by risehr** [comment]
Fixed typos in performance text file.

**Pull request merged**